cmd/daemon/main.go

```diff
@@ -132,55 +140,57 @@ var (
 			},
 			&cli.StringFlag{
 				Name:  flagEthereumEndpoint,
-				Usage: "ethereum client endpoint",
+				Usage: "Ethereum client endpoint",
 			},
 			&cli.StringFlag{
 				Name:  flagEthereumPrivKey,
-				Usage: "file containing a private key hex string",
+				Usage: "File containing a private key as hex string",
 			},
 			&cli.UintFlag{
 				Name:  flagEthereumChainID,
-				Usage: "ethereum chain ID; eg. mainnet=1, ropsten=3, rinkeby=4, goerli=5, ganache=1337",
+				Usage: "Ethereum chain ID; eg. mainnet=1, goerli=5, ganache=1337",
 			},
 			&cli.StringFlag{
 				Name:  flagContractAddress,
-				Usage: "address of instance of SwapFactory.sol already deployed on-chain; required if running on mainnet",
+				Usage: "Address of instance of SwapFactory.sol already deployed on-chain; required if running on mainnet",
 			},
-			&cli.StringFlag{
-				Name:  flagBootnodes,
-				Usage: "comma-separated string of libp2p bootnodes",
+			&cli.StringSliceFlag{
+				Name:    flagBootnodes,
+				Aliases: []string{"bn"},
+				Usage:   "libp2p bootnode, comma separated if passing multiple to a single flag",
 			},
 			&cli.UintFlag{
 				Name:  flagGasPrice,
-				Usage: "ethereum gas price to use for transactions (in gwei). if not set, the gas price is set via oracle.",
+				Usage: "Ethereum gas price to use for transactions (in gwei). If not set, the gas price is set via oracle.",
```

Comment:
Capitalizing or not is a matter of taste.  `monerod` and `geth` both capitalize the first word of flag descriptions (unless it is a proper noun that starts with a lower case letter like `libp2p` above). We had a few places with complete sentences like this one, so I thought capitalizing was a better match.